yogadl/_core.py

```diff
@@ -76,6 +76,7 @@ def stream(
         shuffle_seed: Optional[int] = None,
         shard_rank: int = 0,
         number_of_shards: int = 1,
+        drop_remainder: bool = False,
```

Comment:
Can we name this `drop_shard_remainder` or something?   I personally am much more familiar with "drop remainder" arguments related to batching, and it seems like a good idea to make it impossible to misunderstand.

Reply:
done